tests/regression.py

```diff
@@ -121,6 +124,10 @@ def setUpClass(cls):
 
     @classmethod
     def tearDownClass(cls):
+        with psycopg2.connect("dbname='{}'".format(CONFIG['test_database'])) as conn:
+            with conn.cursor() as cur:
+                if cls.schema:
```

Comment:
You'd want to pull out this 'if' to avoid creating a connection when there is no schema.

Reply:
Fixed.